src/recensio/plone/mails/legacy.py

```diff
@@ -0,0 +1,115 @@
+"""Contains code migrated from the old recensio packages."""
+from html import escape
+from plone import api
+from Products.Five import BrowserView
+from recensio.plone import _
+from recensio.plone.utils import getFormatter
+
+
+def get_formatted_names(
+    full_name_separator, name_part_separator, names, lastname_first=False
+):
+    name_part1 = "firstname"
+    name_part2 = "lastname"
+    if lastname_first:
+        name_part1 = "lastname"
+        name_part2 = "firstname"
+    return escape(
+        full_name_separator.join(
+            [
+                getFormatter(name_part_separator)(x[name_part1], x[name_part2])
+                for x in names
+            ]
+        )
+    )
+
+
+NOTIFICATION_LOG_ADDR = "maillog@recensio.net"
+
+
+class MailNewPublication(BrowserView):
+    """Moved and adapted from recensio.policy.browser.email"""
+
+    def __call__(self):
+        from_name = api.portal.get_registry_record(
+            "plone.email_from_name", default="Recensio"
+        )
+        from_email = api.portal.get_registry_record(
+            "plone.email_from_address", default=""
+        )
+        mail_from = f"{from_name} <{from_email}>"
+
+        referenceAuthors = getattr(self.context, "referenceAuthors", [])
+
+        for author in referenceAuthors:
+            if "email" in author:
+                continue
+            args = {}
+            fuckup = [author["firstname"], author["lastname"]]
```

Comment:
LOL
That name was already present in the old code, but maybe we should take the chance to change this variable name (it's open source after all).